test/evergreen/compatibility_test_for_mongodb_releases.sh

```diff
@@ -0,0 +1,136 @@
+#!/usr/bin/env bash
+##############################################################################################
+# Check releases to ensure forward and backward compatibility.
+##############################################################################################
+
+###########################################################################
+# Return the most recent version of the tagged release.
+###########################################################################
+get_release()
+{ 
+	echo "$(git tag | grep "^mongodb-$1.[0-9]" | sort -n | sed -e '$p' -e d)"
```

Comment:
`sort -n` seems ok to handle 1 decimal digit version numbers in this case. If there are 2 decimal digits available e.g. 3.4.17, it won't query out the latest version. `sort -V` seems working better.